jans-cedarling/cedarling_opa/Dockerfile

```
@@ -0,0 +1,42 @@
+ARG BUILDPLATFORM="linux/amd64"
+ARG BUILDERIMAGE="golang:1.25-trixie"
+ARG JANS_SOURCE_VERSION=c0ae216653f1a615ad053ac1f88f54dfbffd0842
+# Use distroless as minimal base image to package the manager binary
+ARG BASEIMAGE="gcr.io/distroless/cc-debian12:nonroot"
+
+# Stage 1: Rust build
+FROM rust:1.95-slim-bookworm AS rust-builder
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)